ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/encoding/TransactionEncoder.java

```diff
@@ -92,7 +91,7 @@ static void encodeAccessList(final Transaction transaction, final RLPOutput rlpO
     encodeAccessListInner(
         transaction.getChainId(),
         transaction.getNonce(),
-        transaction.getGasPrice(),
+        transaction.getGasPrice().get(),
```

Comment:
Not important but why not use orElseThrow as for maxFeePerGas ?

Reply:
Yup, will change to orElseThrow() on next merge of master